mm2src/mm2_main/src/database/stats_swaps.rs

```diff
@@ -175,6 +182,18 @@ fn insert_stats_maker_swap_sql(swap: &MakerSavedSwap) -> Option<(&'static str, O
     Some((INSERT_STATS_SWAP, params))
 }
 
+fn update_stats_maker_gui_version(swap: &MakerSavedSwap) -> (&'static str, OwnedSqlNamedParams) {
+    let params = owned_named_params! {
+        ":maker_gui": swap.gui.clone(),
+        ":maker_version": swap.mm_version.clone(),
+        ":uuid": swap.uuid.to_string(),
+    };
+    const UPDATE_STATS_SWAP_GUI_AND_VERSION_MAKER: &str =
+        "UPDATE stats_swaps set maker_gui = :maker_gui, maker_version = :maker_version WHERE uuid = :uuid;";
```

Comment:
Just a nit: We can place this const at the start of the function since it is returned first in the tuple, placing consts at the top of functions is also more idiomatic.